Menu/GameShell/10_Settings/Airplane/__init__.py

```diff
@@ -122,7 +122,7 @@ def ScrollUp(self):
             self._Scrolled += dis
 
     def ToggleModeAni(self): ## with animation
-        out = commands.getstatusoutput('sudo rfkill list | grep yes | cut -d " " -f3')
+        out = commands.getstatusoutput('rfkill list | grep yes | cut -d " " -f3')
```

Comment:
could this be configured on the clockworkpi OS image so that sudo doesn't need to be used for this ?

Reply:
Airplane mode needs toggle the rfkill , and it also needs the sudo